README.md

```diff
@@ -1,2 +1,35 @@
-# gonome
+# Gonome
 Genome Edit Design Service
+
+## Goal
+Build a software service for designing genomic edit implementations across therapeutically relevant reference genomes and custom genomes.
+
+### Feature Requirements
+* Store reference whole genome assembly sequences
+* Store custom whole genome assembly sequences
+* Store genome fragment sequences
+* Store reference annotations
+* Store custom annotations
+* Many to many mapping of annotations to sequences
+* Mismatch search for CRISPR guide and primer off targets
+* Saving meta information to guide target database
+* Fundamental DNA calculations (amino acid conversion, binding temp, etc)
+
+### Performance Requirements
+* Calculate an edit design in 1 minute or less
+* Kick off ~20k or more design requests at a time
+
+### Out of Scope
+* Hypothesis generation of edits from functional data
+* Frontend visualization
+* Calculating editing efficiency (can look up in stored metadata)
+
+### Technical Decisions
+* Golang as core implementation language
```

Comment:
at the risk of giving up an A+ repository name, I would like to have some discussion and even a bakeoff around this decision. It's such a foundational component that it feels right be thorough here.

Reply:
Fair enough. It now reads "To be decided by benchmarking". I'll try not to let the current repo name bias the benchmarking approach.